src/components/test/Login.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Login from "../Login";
+import { shallow } from "enzyme";
+import { getPrefabSocialAddress } from "../../test/testAddresses";
+
+const mockSocialAddress = getPrefabSocialAddress(0);
+
+describe("Login", () => {
+  it("renders without crashing", () => {
+    expect(() => {
+      shallow(
+        <Login
+          logout={jest.fn}
+          onAuthenticate={jest.fn}
+          socialAddress={mockSocialAddress}
+        />
+      );
+    }).not.toThrow();
```

Comment:
do we want to test anything else or is it too much of a pain in the butt for an error that would be very obvious